collections/frozenSmslogs.js

```diff
@@ -0,0 +1,94 @@
+// This is a copy of SMSlogs used to track frozen messages.
```

Comment:
Are these meant to appear in the /conversations page, and get counted in reporting, and a bunch of other things that SMSlogs are used for? If so, there are a heap of places we need to update to account of this new collection. For that reason, if these are supposed to be tracked in the same way smslogs are tracked, I think making a new collection is more risky than adding a flag to the old one. But if these aren't being tracked in the same way as smslogs, I agree, a new collection is a good solution.

Reply:
That's an interesting question - my understanding is that we want these to show up in triage to gather data, but in general, these will create proper smslogs when they're actually sent.

If a message ends up not being sent (because an admin replies first), a frozen message shouldn't show up for most purposes; it wasn't actually sent, so for most purposes, doesn't exist.  But it would be good to keep it around to track that the message was frozen (but never sent).

So my thinking is that, for most purposes, these are invisible, and adding a flag on smslogs would require *more* changes to places where they should be ignored.  Does that make sense?